static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -124,13 +125,27 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
   'replay.mutations': (frame: MutationFrame) => ({
     color: 'yellow300',
     description: frame.data.limit
-      ? t(
-          'A large number of mutations was detected (%s). Replay is now stopped to prevent poor performance for your customer.',
-          frame.data.count
+      ? tct(
+          'A large number of mutations was detected [count]. Replay is now stopped to prevent poor performance for your customer. [link]',
+          {
+            count: frame.data.count,
+            link: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/session-replay/configuration/#mutation-limits">
+                {t('Read the docs.')}
```

Comment:
"Read the docs" seems a bit too "RTFM" -- what about something like "Learn more" or "Learn more from our docs" or "Read more from our docs"

Reply:
the tooltip is actually the content in the row but overflowed!

Reply:
and i added the `isHoverable` property to the tooltip